libraries/app/api_objects.cpp

```diff
@@ -69,8 +72,12 @@ market_ticker::market_ticker(const market_ticker_object& mto,
 
    if(!orders.asks.empty())
       lowest_ask = orders.asks[0].price;
+      lowest_ask_base_size =orders.asks[0].base;
```

Comment:
To be consistent with the code already there please add space after `=` here and in the others assignations.

As @abitmore mentioned the 2 `if` blocks need `{` or they will not work as expected. Rest looks good to me.